.commitlintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Yeah, I think you are right. On the other hand, I feel like it may break SRP in a way. Like usually package.json is responsible for quite a lot of stuff anyway (scripts, dependencies, module system, license etc...), so I usually prefer if I can extract smth from there - I prefer to extract. Another benefit that I can see from extraction - is indication between packages - like we don't have to think how big should be config to extract it - there is only one way of doing things, and this usually make it easier to perceive. And third idea that comes to my mind regarding this is that when I see separate files in the repo in file system - I can immediately understand which tools have been setup already. 
The drawback of this is a lot of files in root dir, but I can stand that. 
Lemme know what you think about it - this is not my principal position, but I found it useful to agree on one way to approach this between repos.